variables.tf

```diff
@@ -11,6 +11,14 @@ variable "cluster_name" {
   type = string
 }
 
+variable "cluster_endpoint_public_access" {
+  type = bool
```

Comment:
We should set a default here so that it isn't strictly required to assign this variable a value under normal conditions. It's meant as an override of the default.